commercetools-models/src/main/java/io/sphere/sdk/extensions/ExtendedResourceType.java

```diff
@@ -0,0 +1,23 @@
+package io.sphere.sdk.extensions;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ExtendedResourceType {
```

Comment:
Why didn't you used our `SphereEnumeration` interface as a base? Then you wouldn't have to manually add the mapping logic here 😉

Reply:
Actually with sphere enumeration, you have to start with an uppercase, and i can't request that from the backend since for them its a resource id, its just further safety for us to make it an enum.

Reply:
I see and then I'm fine with it 👍  May be we can add a comment that we can't use `SphereEnumeration` here together with your explanation?

Reply:
Sure, we definitely should keep track 👍